infer/src/dotnet/ResourceLeaksCS.ml

```diff
@@ -62,7 +62,7 @@ module TransferFunctions (CFG : ProcCfg.S) = struct
     in
     match instr with
     | Call (_return, Direct callee_procname, HilExp.AccessExpression allocated :: _, _, _loc)
-      when acquires_resource tenv callee_procname && is_not_enumerable ->
+      when acquires_resource tenv callee_procname && is_not_enumerable->
```

Comment:
Please try to run `make fmt` before submitting PRs to make reviewing easier and so that the CI doesn't fail because of ill-formatted code.

Reply:
Done